demisto_sdk/commands/generate_docs/generate_integration_doc.py

```diff
@@ -421,3 +420,27 @@ def command_permissions_filter(permission):
     elif permission.startswith('!'):
         return f'{permission}'
     return permission
+
+
+def add_access_data_of_type_credentials(access_data: List[Dict], credentials_conf: Dict) -> None:
+    """
+    Adds to 'access_data' the parameter data of credentials configuration parameter.
+    Args:
+        access_data (List[Dict]): Access data to add the credentials conf data to.
+        credentials_conf (Dict): Credentials conf data.
+
+    Returns:
+        (None): Adds the data to 'access_data'.
+    """
+    display_name = credentials_conf.get('display')
+    display_password_name = credentials_conf.get('displaypassword', 'Password')
+    if display_name:
+        access_data.append(
+            {'Parameter': display_name,
+             'Description': string_escape_md(credentials_conf.get('additionalinfo', '')),
+             'Required': credentials_conf.get('required', '')})
+    if display_password_name:
```

Comment:
this is always true

Reply:
can we have both of the fields? make sure we don't append twice

Reply:
@dantavori 
Removed the if, anyway if both fields exist it should have 2 lines in the Parameter configuration, see this
https://github.com/demisto/etc/issues/37184